### PR TITLE
Testing `@ctpps` DQM sequence on 2018 ZeroBias dataset

### PR DESCRIPTION
**Pull request by francescobrivio**
# Replay Request

**Requestor**  
Francesco Brivio for PPS/DQM

**Describe the configuration**  
* Release: CMSSW_12_2_1_patch2
* Run: 324841
* GTs:
   * expressGlobalTag: 122X_dataRun3_Express_TIER0_REPLAY_Run2_v1
   * promptrecoGlobalTag: 122X_dataRun3_Prompt_TIER0_REPLAY_Run2_v1
   * alcap0GlobalTag: 122X_dataRun3_Prompt_TIER0_REPLAY_Run2_v1
* Additional changes:
   * Removed all the PDs except for ZeroBias (to save computing resources)
   * Ignored all the express streams (to save computing resources)
   * Set correct eras for running on 2018 data
   * Added `"@ctpps"` DQM sequence to ZeroBias dataset

**Purpose of the test**  
This replay is to test the `"@ctpps"` DQM sequence and conclude #4588 that was never followed up from last year.
I used the configuration from the PPS replay on 2018 data (#4640) which was successful, and only added the new DQM sequence. @germanfgv @jhonatanamado please check the whole config and especially if the `ignoreStream`s are correctly used.

FYI @vavati @fabferro @jfernan2 @tvami @malbouis 

**T0 Operations cmsTalk thread**  
N/A

### CODE REVIEW

File: etc/ReplayOfflineConfiguration.py [hunk 1]
```diff
@@ -1563,6 +561,37 @@
...
+ignoreStream(tier0Config, "ALCALUMIPIXELSEXPRESS")
+ignoreStream(tier0Config, "ALCAP0")
+ignoreStream(tier0Config, "ALCAPHISYM")
+ignoreStream(tier0Config, "Calibration")
+ignoreStream(tier0Config, "Express")
+ignoreStream(tier0Config, "ExpressAlignment")
+ignoreStream(tier0Config, "ExpressCosmics")
+ignoreStream(tier0Config, "ExpressPA")
+ignoreStream(tier0Config, "HIExpress")
+ignoreStream(tier0Config, "HIExpressAlignment"
```
Comment (jhonatanamado) [inline etc/ReplayOfflineConfiguration.py:577]: Hi @francescobrivio here is the syntax error, you forgot ")"

File: etc/ReplayOfflineConfiguration.py [hunk 2]
```diff
@@ -1563,6 +561,37 @@
...
+ignoreStream(tier0Config, "HLTMonitor")
+ignoreStream(tier0Config, "HLTMonitorPA")
+ignoreStream(tier0Config, "NanoDST")
+ignoreStream(tier0Config, "ParkingBPH1")
+ignoreStream(tier0Config, "ParkingBPH2")
+ignoreStream(tier0Config, "ParkingBPH3")
+ignoreStream(tier0Config, "ParkingBPH4")
+ignoreStream(tier0Config, "ParkingBPH5")
+ignoreStream(tier0Config, "PhysicsCommissioning")
```
Comment (jhonatanamado) [inline etc/ReplayOfflineConfiguration.py:586]: The streamer `PhysicsCommissioning` needs to be included in order to generate PDs such us `ZeroBias`

**Review by jhonatanamado — CHANGES_REQUESTED** [review]

File: etc/ReplayOfflineConfiguration.py (hunk 2 above)
Comment (francescobrivio) [reply etc/ReplayOfflineConfiguration.py:586]: ah ok, makes sense! i'll remove this line

**Review by jhonatanamado — APPROVED** [review]

**Comment by jhonatanamado** [comment]
run replay please

**Comment by francescobrivio** [comment]
Just to add more to the fact that the replay finished succesfully, here is the link to the DQM plots generated: https://tinyurl.com/y9z68y22

@vavati @fabferro @jfernan2 please provide feedback on the DQM plots.

**Comment by francescobrivio** [comment]
The change tested here has been implemented in #4655

**Pull request closed without merging**